spec/models/user_spec.rb

```diff
@@ -1,10 +1,26 @@
 require 'spec_helper'
 
 describe User do
+  around(:each)do
+   |example| ActionMailer::Base.deliveries.clear
```

Comment:
`|example|` should be on the same line as the previous `do` and immediately after it, i.e. `do |example|`